### DISCUSSION TITLE
Why FMI2.0 and FMU type is CoSimulation can't be import

### DISCUSSION THREAD

**Discussion by lijunliangTG** [OpenModelica/OpenModelica#10414] [Q&A]
I want to know how to achieve function import FMI2.0 and FMU type is CoSimulation. I trace the file to import fmu files  OpenModelicaScriptingAPI , but i cant't understand how this is achieved the function.

I have the following questions：
1、Where is the function implementation for importing fmu? How can I understand them?
2、How can I modify the fmu 2 import file to get him to support FMI2.0 and CoSimulation from other platforms like Dymola ?

**Comment by adeas31** [comment]
We don't support importing FMI 2.0 co-simulation as Modelica code and there are no plans to support it either.
Use [OMSimulator](https://github.com/OpenModelica/OMSimulator) instead.

**Comment by casella** [comment]
Modelica requires the _equations_ of the model. This is (kind of) possible with model exchange, that contains the hybrid state-space equations of the system, not with co-simulation, that also contains an ODE solver.
